test/OpenTelemetry.Instrumentation.AspNet.Tests/HttpInListenerTests.cs

```diff
@@ -177,32 +164,33 @@ public void Dispose()
                     {
                         options.Enrich = GetEnrichmentAction(default);
                     }
+
+                    options.RecordException = recordException;
                 })
             .AddProcessor(activityProcessor.Object).Build())
```

Comment:
Is Mock easier/better than InMemoryExporter ?

Reply:
“It shines, somewhere, you do not see it” -Kung Fu 🦗 (there's no grasshopper emoji, which is upsetting)

I'm not a fan of using mocking anywhere, TBH. Prefer to always make an API that is testable. My guess would be this test was written before InMemoryExporter existed? In any case, I cleaned it up a bit.